examples/calibrate_CATKE_to_LESbrary/visualize_profile_predictions.jl

```diff
@@ -66,7 +66,7 @@ function visualize!(ip::InverseProblem, parameters;
             truth = getproperty(data, field)
             prediction = getproperty(predictions_batch[i], field)
 
-            z = field ∈ [:u, :v] ? data.grid.zF[1:data.grid.Nz] : data.grid.zC[1:data.grid.Nz]
+            z = field ∈ [:u, :v] ? data.grid.zᵃᵃᶠ[1:data.grid.Nz] : data.grid.zᵃᵃᶜ[1:data.grid.Nz]
```

Comment:
should we use `znodes` here?

Reply:
sure, why not

Reply:
```Julia
z = field ∈ [:u, :v] ? znodes(Face, data.grid)[1:data.grid.Nz] : znodes(Center, data.grid)[1:data.grid.Nz]
```
?

Reply:
why was it `data.grid.zF[1:data.grid.Nz]` and not `data.grid.zF[1:data.grid.Nz+1]`?

Reply:
@glwagner, shall I change this or merge?

Reply:
I'll merge... This is anyway in a visualisation script in the CATKE folder... We'll have a chance to revisit this!